covid19dp_submission/VERSION

```diff
@@ -1 +1 @@
-0.1.dev1
\ No newline at end of file
+0.1.1
```

Comment:
Shall we do tagging as a separate commit to the main branch (similar to our [deployment process](https://www.ebi.ac.uk/seqdb/confluence/display/VAR/Release+and+deployment+-+SOP) for Java projects)?